common/src/main/java/com/hedera/block/common/utils/Preconditions.java

```diff
@@ -185,6 +185,50 @@ public static long requireGreaterOrEqual(final long toTest, final long base, fin
         throw new IllegalArgumentException(message);
     }
 
+    /**
+     * This method asserts a given int is within a range (boundaries
+     * included). If the given int is within the range, then we return it,
+     * else, an {@link IllegalArgumentException} is thrown.
+     *
+     * @param toCheck the int value to test
+     * @param lowerBoundary the lower boundary
+     * @param upperBoundary the upper boundary
+     * @return the input {@code toCheck} if it is within the range (boundaries
+     * included)
+     * @throws IllegalArgumentException if the input int does not pass the test
+     */
+    public static int requireInRange(final int toCheck, final int lowerBoundary, final int upperBoundary) {
+        return requireInRange(toCheck, lowerBoundary, upperBoundary, null);
```

Comment:
I agree with the idea of replacing the `null` parameter with a default value.

Reply:
Will be addressed in a chore a bit later.